sdk/android/build.gradle

```diff
@@ -53,7 +53,7 @@ allprojects {
     }
 }
 
-project.ext.libraryVersion = 'v4.44.0_RC2'
+project.ext.libraryVersion = 'v4.44.0_RC8'
```

Comment:
Shall we use the latest version?

Reply:
that last release version, don't bother about it , it will be changed every time jenkins build new RC